master/worker.go

```diff
@@ -16,16 +17,21 @@ type Worker struct {
 	client proto.WorkerClient
 }
 
+// Connect to worker server running on paticular ip and port
 func (w *Worker) Connect() error {
-	address := fmt.Sprintf("%s:%d", w.ip, w.port)
-	conn, err := grpc.Dial(address, grpc.WithInsecure())
+	fmt.Println(w)
+	conn, err := rpc.
+		NewClientConnBuilder(w.ip, w.port).
+		Build()
 	if err != nil {
-		return err
+		panic(err)
```

Comment:
We should return error ( handle it in the caller ) instead of panic ( crashing the server )